content/en/docs/guides/reflection.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Reflection
+description: >-
+  Explains how reflection can be used to improve the transparency and
+  interpretability of RPCs.
+---
+
+### Overview
+
+Reflection is [a
+protocol](https://github.com/grpc/grpc-proto/blob/master/grpc/reflection/v1/reflection.proto)
+that gRPC servers can use to declare the APIs they export over a standardized
+RPC service, including all types referenced by the request and response
+messages. Clients can then use this information to encode requests and decode
+responses in human-readable manner.
+
+Reflection is used heavily by debugging tools such as
+[`grpcurl`](https://github.com/fullstorydev/grpcurl) and
+[Postman](https://learning.postman.com/docs/sending-requests/grpc/grpc-client-overview/).
+One coming from the REST world might compare the gRPC reflection API to serving
+an OpenAPI document on the HTTP server presenting the REST API being described.
+
+### Transparency and Interpretability
+
+A big contributor to gRPC's stellar performance is the use of Protobuf for
+serialization -- a _binary_ non-human-readable protocol. While this greatly
+speeds up an RPC, it can also make it more difficult to manually interact with a
+server. In order to manually send a gRPC request to a server over HTTP/2 using
+`curl`, you would have to:
```

Comment:
Maybe add "Hypothetically, " at the beginning of this sentence to make it clear this si not actually something anyone should attempt.

Reply:
Done.